content/en/internal_developer_portal/software_catalog/entity_model/entity_types.md

```diff
@@ -401,6 +401,50 @@ Learn more about [peer tags and inferred entities][4].
 
 {{% /tab %}}
 
+{{% tab "Frontend" %}}
+In Software Catalog, a frontend (`kind:frontend`) represents a frontend application—such as a browser-based SPA or mobile app—that interacts with services and APIs. Frontend entities offer a structured way to model user-facing applications in the same catalog alongside backend services.
+
+### Example YAML definitions
+```
```

Comment:
This will enable syntax highlighting and the Copy button in the rendered docs:
```suggestion
```yaml
```